src/TreeMenu/renderProps.tsx

```diff
@@ -73,7 +79,7 @@ export const defaultChildren: TreeMenuChildren = ({ search, items }) => {
   const onSearch = (e: React.ChangeEvent<HTMLInputElement>) => {
     const { value } = e.target;
     search && search(value);
-  };
+  };  
```

Comment:
Did you use prettier? There're some random white space here and there. Maybe you should activate it on your IDE.